cli/start.ts

```diff
@@ -16,10 +17,11 @@ Options:
 
 export default async function (appDir: string, options: Record<string, string | boolean>) {
     const { start } = await import('../server.ts')
+    const host = String(options.h || options.host || 'localhost')
```

Comment:
```suggestion
    const host = String(options.hn || options.hostname || 'localhost')
```

Reply:
or `-H, --host`, i want to keep it simple, what do you think?

Reply:
We could also avoid a shorthand altogether, `--host` should work fine for people.